tests/pubsub/testgossipsub.nim

```diff
@@ -753,16 +763,14 @@ suite "GossipSub":
 
     await allFuturesThrowing(nodes.mapIt(allFutures(it.switch.stop())))
 
-    await allFuturesThrowing(nodesFut)
-
   asyncTest "e2e - GossipSub with multiple peers (sparse)":
     var runs = 10
 
     let
       nodes = generateNodes(runs, gossip = true, triggerSelf = true)
       nodesFut = nodes.mapIt(it.switch.start())
```

Comment:
it would be nice to refactor these line, across all tests similarly how `testgossipsubparameters` file use it...

Reply:
talking about: `nodesFut = nodes.mapIt(it.switch.start())`